app/main.py

```
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        if comfort_class < 1 or comfort_class > 7:
+            print("Please enter the number between 1 and 7")
+        if clean_mark < 1 or clean_mark > 10:
+            print("Please enter the number between 1 and 10")
```

Currently, if comfort_class or clean_mark are out of range, the code only prints a message but still allows the creation of an invalid Car object. According to typical class design and likely checklist expectations, you should raise an exception (e.g., ValueError) instead of just printing, to prevent invalid objects from being created.